validation/superchain-config_test.go

```diff
@@ -20,12 +20,27 @@ func testSuperchainConfig(t *testing.T, chain *ChainConfig) {
 
 	client, err := ethclient.Dial(rpcEndpoint)
 	require.NoErrorf(t, err, "could not dial rpc endpoint %s", rpcEndpoint)
-	opp := Addresses[chain.ChainID].OptimismPortalProxy
 
-	got, err := getAddress("superchainConfig()", opp, client)
-	require.NoError(t, err)
+	checkSuperchainConfig(t, client, Addresses[chain.ChainID].OptimismPortalProxy, *expected)
+	checkSuperchainConfig(t, client, Addresses[chain.ChainID].AnchorStateRegistryProxy, *expected)
+	checkSuperchainConfig(t, client, Addresses[chain.ChainID].L1CrossDomainMessengerProxy, *expected)
+	checkSuperchainConfig(t, client, Addresses[chain.ChainID].L1ERC721BridgeProxy, *expected)
+	checkSuperchainConfig(t, client, Addresses[chain.ChainID].L1StandardBridgeProxy, *expected)
```

Comment:
Nit: Do you think it would be worth enshrining more of this information into the standard config toml files? I think there is precedent for declaring that `contractFoo.MethodBar` should resolve to `AddressBaz` already.

Reply:
Since changing those toml files requires a governance approval, I think we should hold off on enshrining this change for now